modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7012,9 +7013,11 @@ feature.flag.LPS-135430.title=Proxy Object
 feature.flag.LPS-179669.description=Search the company index for matching content. Add facets with translated display terms and set a search blueprint in the request to add custom searching, filtering, aggregation, and sorting.
 feature.flag.LPS-179669.title=Search Headless API
 feature.flag.type.description.beta=Beta features are experimental and not supported. Use with caution.
+feature.flag.type.description.deprecation=Deprecated features will be removed from DXP soon. They might be replaced with a new and better alternative.
 feature.flag.type.description.dev=Developer features are under active development. Do not activate unless you are on a development instance of DXP.
 feature.flag.type.description.release=Release features provide an easy way to onboard new functionality in DXP. They are fully supported and can be activated individually and progressively.
 feature.flag.type.title.beta=Beta
+feature.flag.type.title.deprecation=Deprecated
```

Comment:
The translation should be "Deprecation".

Reply:
When talking to Mateo, he said it should be `Deprecated`. That was last month when I started this story. Either that changed or he just missed that. I'll change it. Thank you.

Reply:
My bad

Reply:
Nope, the description should still be "Deprecated features". Maybe this will help... When we deprecate a feature, we use a Deprecation feature flag to toggle the visibility of the deprecated feature.